index.html

```diff
@@ -86,13 +86,23 @@
             button.onclick = weather;
         }
     </script>
+    <!-- Leaflet CSS for OpenStreetMap -->
```

Comment:
CSS from leaflet - these styles would need to be overridden or this could be removed to apply custom styles